src/microsoft/1.18/windows/nanoserver-1809/Dockerfile

```diff
@@ -0,0 +1,36 @@
+#
+# NOTE: THIS DOCKERFILE IS GENERATED VIA "apply-templates.sh"
+#
+# PLEASE DO NOT EDIT IT DIRECTLY.
+#
+
+ARG REPO=mcr.microsoft.com/oss/go/microsoft/golang
+# It's easy to download Go and install it in server core, but not nanoserver. So, to build
+# nanoserver, copy over the server core installation.
+ARG DOWNLOADER_TAG=1.18.0-windowsservercore-1809
+FROM $REPO:$DOWNLOADER_TAG AS downloader
+
+FROM mcr.microsoft.com/windows/nanoserver:1809
+
+SHELL ["cmd", "/S", "/C"]
+
+# no Git installed (intentionally)
+#  -- Nano Server is "Windows Slim"
+
+# for 1.17+, we'll follow the (new) Go upstream default for install (https://golang.org/cl/283600), which frees up C:\go to be the default GOPATH and thus match the Linux images more closely (https://github.com/docker-library/golang/issues/288)
+ENV GOPATH C:\\go
```

Comment:
The difference in case for `c:\go` and `Go` on all other paths here continues to be a head scratcher for me 😄

Reply:
It probably would be `C:\Go`, but this is matching the Linux default, and on Linux you barely ever capitalize paths. So only the Windows-specific parts of the Dockerfile get the uppercase treatment.